build.sbt

```diff
@@ -345,13 +310,14 @@ lazy val buildpressConfig = (project in file("buildpress-config"))
   )
 
 lazy val buildpress = project
-  .dependsOn(bloopgun, bloopShared, buildpressConfig)
```

Comment:
This was the sole reason that bloopgun was depended by from frontend and it was only due to usage of one method in one class (Shell) :sweat_smile: 

Now we don't have to release bloop rifle for 2.12 at all. (we could, but that's not needed now)